app/Gruntfile.js

```diff
@@ -5,9 +5,7 @@ module.exports = function(grunt) {
     pkg: '<json:package.json>',
     jshint: {
       all: {
-        src: [
-          'js/index.js', 'js/main.js'
-        ]
+        src: ['js/*.js']
```

Comment:
This doesn't test hterm.

Here's the result. only two file is tested.  Do you think we have to specify each file?

$ grunt test
Running "jshint:all" (jshint) task
$$ 2 files lint free